authentication/pom.xml

```diff
@@ -1,16 +1,22 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+  ~ one or more contributor license agreements. See the NOTICE file distributed
+  ~ with this work for additional information regarding copyright ownership.
+  ~ Licensed under the Camunda License 1.0. You may not use this file
+  ~ except in compliance with the Camunda License 1.0.
+  -->
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>io.camunda</groupId>
-    <artifactId>identity-parent</artifactId>
+    <artifactId>zeebe-parent</artifactId>
     <version>8.6.0-SNAPSHOT</version>
-    <relativePath>../pom.xml</relativePath>
+    <relativePath>../parent/pom.xml</relativePath>
   </parent>
+  <artifactId>camunda-authentication</artifactId>
 
-  <artifactId>auth-spring-boot-starter</artifactId>
-
-  <name>Identity Auth Spring Boot Starter</name>
+  <name>Camunda Authentication</name>
```

Comment:
@Ben-Sheppard, a question for my understanding:
When going through the code of this module, I noticed that it uses the `JdbcUserDetailsManager` from Spring. To me, it feels like a Spring lock-in because they expect a specific DB schema, etc.

How flexible is it for extension? What happens if they change something that requires a different schema? A typical feature request from users is to configure table prefixes, would this still be possible with that approach?

(I know this is not related to this PR itself, but would be great, if you could guide me through it soon.)

Reply:
Hi @romansmirnov - thats correct, we elected to take the approach of aligning closely with Spring to reduce the amount of custom code required. As things currently are we can override the things we need to but fallback to the core for other things.

Having a different schema I accept at the moment is an issue however longer term I believe we should look for something like Flyway, this way we could manage the schemas and handle migrations when required. I hadn't heard of a configurable table prefix feature request before so this could well mean we need to rethink things.

My preference is that I'd like to keep it as it is for now to keep progressing but take the discussion to a separate context.